lib/internal/util/comparisons.js

```diff
@@ -132,25 +123,7 @@ function isIdenticalTypedArrayType(a, b) {
   if (check !== undefined && check(a)) {
     return check(b);
   }
-  // Manipulated Symbol.toStringTag.
-  for (const check of [
-    isUint16Array,
-    isUint32Array,
-    isInt8Array,
-    isInt16Array,
-    isInt32Array,
-    isFloat32Array,
-    isFloat64Array,
-    isBigInt64Array,
-    isBigUint64Array,
-    isUint8ClampedArray,
-    isUint8Array
-  ]) {
-    if (check(a)) {
-      return check(b);
-    }
-  }
-  /* c8 ignore next 4 */
+  /* c8 ignore next 5 */
```

Comment:
It would actually be sufficient to replace the function content with:

```js
return TypedArrayPrototypeGetSymbolToStringTag(a) === TypedArrayPrototypeGetSymbolToStringTag(b);
```

I would probably inline the function.

Reply:
Sure, inlined.